modules/storage.js

```diff
@@ -0,0 +1,26 @@
+export function get(callback) {
+  chrome.storage.sync.get(callback)
+}
+
+export function set(page) {
+  chrome.storage.sync.set(page)
+}
+
+export function remove(time) {
+  chrome.storage.sync.remove(time)
+}
+
+export function clear() {
+  chrome.storage.sync.clear()
+}
+
+export function setUnique(page) {
+  get((pages) => {
+    if (!has(pages, page)) set(page)
```

Comment:
Suggestion for prototype: `!pages.has(page)` would be easier to read.
Or `!page.in(pages)`